backend/StreamServerApp/subtitles.py

```diff
@@ -16,21 +16,49 @@ def init_cache():
             'filename': 'cachefile.dbm'}, replace_existing_backend=True)
 
 
-def handle_subliminal_download(video, video_path, langage):
-    best_subtitles = download_best_subtitles([video], {Language(langage)})
+def handle_subliminal_download(video, video_path):
+    """ # Download the best subtitles in french and english
+        Args:
+        video : Name of video
+        video_path: absolute path to videos
+    """
+    best_subtitles = download_best_subtitles([video], {Language('fra'),Language('eng')})
     if best_subtitles[video]:
-        best_subtitle = best_subtitles[video][0]
-        value = save_subtitles(video, [best_subtitle], encoding='utf8')
-        if len(value) > 0:
+
+        if best_subtitles[video][0].language == Language('eng') :
+            best_subtitle_eng = best_subtitles[video][0]
+            best_subtitle_fr = best_subtitles[video][1]
+        else : 
+            best_subtitle_fr = best_subtitles[video][0]
+            best_subtitle_eng = best_subtitles[video][1]
+
+        value_fr = save_subtitles(video, [best_subtitle_fr], encoding='utf8')
+        value_eng = save_subtitles(video, [best_subtitle_eng], encoding='utf8')
+        webvtt_fullpath=[]
+
+        if len(value_fr) > 0:
+            srt_fullpath = subtitle.get_subtitle_path(
+                video_path, Language('fra'))
+            webvtt_fr_fullpath = os.path.splitext(srt_fullpath)[0]+'.vtt'
+            if(os.path.isfile(webvtt_fr_fullpath) is True):
+                #return subtitles path even if subtitles are already downloaded/converted
+                webvtt_fullpath.append(webvtt_fr_fullpath)
+            if(os.path.isfile(srt_fullpath)): 
+                convert_subtitles_to_webvtt(srt_fullpath, webvtt_fr_fullpath)
+                webvtt_fullpath.append(webvtt_fr_fullpath)
```

Comment:
This should be simplified or clarified somehow.
Also, if you do with the iteration as proposed in a comment before, you can avoid repeating this part twice and do things like:
```python
            srt_fullpath = subtitle.get_subtitle_path(
                video_path, retrieved_subtitle.language)
```